### DISCUSSION TITLE
cesium point clouds disappearing

### DISCUSSION THREAD

**Discussion by radoslavirha** [visgl/deck.gl#5776] [General]
Hi,

I'm facing an issue with Cesium point cloud  (Tile3DLayer + Tiles3DLoader). Sometimes most of the points disappear and after a small move with the mouse I can see detailed point cloud again. So I think the data is correct??. Is there any parameter in deck.gl or loaders.gl to fix this issue? I thought it can be refinement in 3d tiles. I set it in the root tile ('REPLACE', 'ADD') but without any change. 

```
new Tile3DLayer({
        id: layer.label,
        data: `${api_url}/tileset.json`,
        loader: Tiles3DLoader,
        pointSize: layer.pointSize || 1
})
```

At the beginning of the video you can see only few points (and it's flickering sometimes).

https://user-images.githubusercontent.com/23722745/118234282-6c979e00-b493-11eb-9022-b90900a00317.mov

Any ideas what is wrong, or where can I look for a bug?

Thanks!

**Comment by ibgreen** [comment]
It is a bit hard to say but I suspect that this flickering happens when you move between two tiles in your 3D Tiles tileset.
The logic behind dynamic tile selection (traversal) and dynamic loading of tiles is handled by loaders.gl. Can you open a bug there with this information?

@belom88 @atomicra
